Guide.docc/RuntimeBehavior.md

```diff
@@ -0,0 +1,207 @@
+# Runtime Behavior
+
+
+Learn how Swift concurrency runtime semantics differ from other runtimes you may 
+be familiar with, and familiarize yourself with common patterns to achieve 
+similar end results in terms of execution semantics.
+
+Swift's concurrency model with a strong focus on async/await, actors and tasks,
+means that some patterns from other libraries or concurrency runtimes don't 
+translate directly into this new model. In this section, we'll explore common 
+patterns and differences in runtime behavior to be aware of, and how to address 
+them while you migrate your code to Swift concurrency.
```

Comment:
I think `runtime behaviour` is a great article name but IMO we should talk about executors and executor jobs here first and how they relate to `Tasks` before we dive into implementing a specific pattern of ordered processing.

Reply:
Sure, I'm not writing those in "final document order";

we'll do https://github.com/apple/swift-migration-guide/issues/59 which will explain executors, and it will be here; no worries.